packages/expo-build-properties/src/pluginConfig.ts

```diff
@@ -338,6 +338,17 @@ export interface PluginConfigTypeIos {
    * and [Apple's documentation on Privacy manifest files](https://developer.apple.com/documentation/bundleresources/privacy_manifest_files).
    */
   privacyManifestAggregationEnabled?: boolean;
+
+  /**
+   * Enable support for prebuilt React Native iOS dependencies (ReactNativeDependencies.xcframework).
+   * This feature is only available from React Native 0.80.
+   * `false` will set `ENV['RCT_USE_RN_DEP'] = '1'` in the Podfile to use prebuilt third-party dependencies.
+   *
+   * @default true
+   * @see [RN documentation on prebuilt dependencies](https://reactnative.dev/blog/2025/06/12/react-native-0.80#experimental---react-native-ios-dependencies-are-now-prebuilt)
```

Comment:
```suggestion
   * @see React Native documentation on [prebuilt dependencies](https://reactnative.dev/blog/2025/06/12/react-native-0.80#experimental---react-native-ios-dependencies-are-now-prebuilt) for more information.
```

We avoid using abbreviations in our docs. Let's update this for consistency.